openpype/hosts/blender/plugins/publish/integrate_blend.py

```diff
@@ -1,8 +1,12 @@
+import asyncio
+import gc
 from subprocess import Popen
+from time import sleep
```

Comment:
Maybe these are test imports ? Can we clean unused import ?

Reply:
Right